Dockerfile

```diff
@@ -1,21 +1,21 @@
-FROM golang:1.17.3-alpine3.14 as builder
+FROM golang:1.17-alpine3.15 as builder
 
 WORKDIR /go/src/github.com/rajatjindal/krew-release-bot
 COPY . .
 
+RUN apk add --no-cache git
```

Comment:
why do we need this?

Reply:
Since the patch removes vendor/ (not sure why) go uses git to fetch.

Reply:
Nice catch, I should remove this line after reverting change to vendor/.